infrastructure/lib/constructs/robomo-discord-bot.ts

```diff
@@ -20,12 +20,10 @@ export class RobomoDiscordBot extends Construct {
     const cluster = new ecs.Cluster(this, 'DiscordBotFargateCluster');
 
     const imageAsset = new DockerImageAsset(this, 'DiscordBotECSDockerImage', {
-      directory: path.join(__dirname, '../../robomo-discord-bot'),
+      directory: path.join(__dirname, '../../../robomo-discord-bot'),
     });
 
-    const logGroup = new LogGroup(this, 'Logs', {
-      logGroupName: 'DiscordBotECSLogGroup',
-    });
+    const logGroup = new LogGroup(this, 'Logs');
```

Comment:
any reason not to keep the explicit log group name? sometimes makes it easier to find things in CW

Reply:
oops, I think this was another confusion on my part. I had deleted my whole cdk stack yesterday but hadn't deleted the log group and cdk had complained a log group with that name already existed. will revert this change here

Reply:
you're right though, that's a good lesson - some resources like log groups don't get deleted when the stack gets deleted, and then you will get an error like that when you redeploy, so you have to go delete the log group first before you deploy. but for some resources it's worth it to prevent it from coming up with a name like `EcsDiscoEcsLAJJREERARARARRRARA` :)